packages/FormElement/src/components/Instructions/Instructions.js

```diff
@@ -1,21 +1,36 @@
 import React from "react";
 import PropTypes from "prop-types";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Heading from "@paprika/heading";
 import * as sc from "./Instructions.styles";
 
 const propTypes = {
   children: PropTypes.node.isRequired,
+  hasTitle: PropTypes.bool,
+};
+
+const defaultProps = {
+  hasTitle: false,
 };
 
 function Instructions(props) {
-  const { children, ...moreProps } = props;
+  const { children, hasTitle, ...moreProps } = props;
+  const I18n = useI18n();
 
   return (
     <sc.Instructions data-pka-anchor="form-element.instructions" {...moreProps}>
+      {hasTitle ? (
+        <Heading level={3} displayLevel={6} isLight>
+          {I18n.t("formElement.instructions")}
+        </Heading>
+      ) : null}
```

Comment:
Looks good to me! Isn't this already supported by `< Instructions />` component? Is this one of the design patterns?

Reply:
I mean developers can just pass the `<Heading>Instructions</Heading>` in the children of `< Instructions >`

Reply:
Not sure what you mean, currently there are no mocks for this sub component, and the component didn't have a title,  but I think it makes sense to include tbh. I don't really want to have the consumer start passing it it html in order to achieve this and we don't want to repeat that in all of our consuming applications

Reply:
Yes, we could but I don't think it's necessary for the consumer to do it if we can help it, I would think the title belongs in this component

Reply:
Perhaps we could allow customization of that title, or have the consumer pass it as a child, but i think just for convenience we can have the formElement component take of it